code-of-conduct.html

```diff
@@ -40,8 +40,8 @@
 
       <nav id="navbar" class="navbar order-last order-lg-0">
         <ul>
-          <li><a class="nav-link scrollto" href="/index.html">Home</a></li>
-          <li><a class="nav-link scrollto active" href="/code-of-conduct.html">Code of Conduct</a></li>
+          <li><a class="nav-link scrollto" href="index.html">Home</a></li>
+          <li><a class="nav-link scrollto active" href="#">Code of Conduct</a></li>
```

Comment:
@basiima what is the essence of this change? why are we changing from `href="/code-of-conduct.html"` to `href="#"`

Reply:
alright that should be okay then